scripts/cherry-pick-commits.js

```diff
@@ -172,15 +185,24 @@ async function run() {
   // Always cherry-pick on top of the last tag.
   // If we used the cherry-picked master commit as base in the case of subsequent patches, we'd inherit
   // non-patch commits from master between the last .0 release and the previous patch release.
-  const results = await attemptCherryPicks(tag, list);
+  const results = await attemptCherryPicks(tag, list, mode);
 
-  console.log('');
-  console.log('Test-running build...');
-  const buildSucceeded = checkSpawnSuccess('npm run build');
+  let buildSucceeded;
+  let testsSucceeded;
+  const skipTests = args.includes('--skip-tests');
 
-  console.log('');
-  console.log('Running unit tests...');
-  const testsSucceeded = checkSpawnSuccess('npm run test:unit');
+  if (skipTests) {
+    console.log('Skipping npm run build and npm run test:unit.');
+    console.log('You should _really_ run these unless you\'re just testing this script.');
```

Comment:
Is the _really_ supposed to appear italicized in the terminal?

Reply:
I think that depends on the terminal emulator being used. I didn't feel like going crazy into formatting codes...